frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -113,22 +113,40 @@ export const countNodesPerZone = (nodes: NodeKind[]) =>
 export const nodesWithoutTaints = (nodes: NodeKind[]) =>
   nodes.filter((node: NodeKind) => hasOCSTaint(node) || hasNoTaints(node));
 
-export const isArbiterSC = (
-  sc: StorageClassResourceKind,
+const getSelectedNodes = (
+  scName: string,
   pvData: K8sResourceKind[],
   nodesData: NodeKind[],
-): boolean => {
-  const pvs: K8sResourceKind[] = getSCAvailablePVs(pvData, getName(sc));
+): NodeKind[] => {
+  const pvs: K8sResourceKind[] = getSCAvailablePVs(pvData, scName);
   const scNodeNames = getAssociatedNodes(pvs);
   const tableData: NodeKind[] = nodesData.filter(
     (node: NodeKind) =>
       scNodeNames.includes(getName(node)) ||
       scNodeNames.includes(node.metadata.labels?.['kubernetes.io/hostname']),
   );
+  return tableData;
+};
+
+export const isIntenalAttachedSC = (
+  scName: string,
+  pvData: K8sResourceKind[],
+  nodesData: NodeKind[],
+): boolean => {
+  const tableData: NodeKind[] = getSelectedNodes(scName, pvData, nodesData);
+  return tableData.length >= MINIMUM_NODES;
```

Comment:
we need to add check for rack labels as well.
This is to make sure that the PVs are exactly from three different racks.
Sorry for the last minute change, but its final.
Thanks.